data/depots_utahtransportationauthority59.csv

```diff
@@ -0,0 +1,5 @@
+lat,lng,name,max_buses,url
+40.762455,-111.911353,"UTA Proposed Depot District Clean Fuels Tech Center",9999,"http://www.slcdocs.com/transportation/Plans/SLC_TMP_FULL_FINAL.pdf"
+40.276880,-111.729939,"UTA Mt. Timpanogos",9999,""
```

Comment:
I modified this with some additional depots you missed and removed the central depot, which is being replaced by the clean fuels depot across the street.

Reply:
thanks. citation?